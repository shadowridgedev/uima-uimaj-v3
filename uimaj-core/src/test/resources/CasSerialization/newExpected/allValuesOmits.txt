{"@context" : {
    "AllTypes" : {
      "@id" : "org.apache.uima.test.AllTypes", 
      "@featureRefs" : ["sofa",  "aFS",  "aArrayMrBoolean",  "aArrayMrByte",  "aArrayMrShort",  "aArrayMrString",  "aListMrInteger",  "aListMrString",  "aListFs",  "aListMrFs" ], 
      "@superTypes" : ["uima.tcas.Annotation",  "uima.cas.AnnotationBase",  "uima.cas.TOP" ] }, 
    "BooleanArray" : {
      "@id" : "uima.cas.BooleanArray", 
      "@superTypes" : ["uima.cas.FSArray",  "uima.cas.ArrayBase",  "uima.cas.TOP" ] }, 
    "ByteArray" : {
      "@id" : "uima.cas.ByteArray", 
      "@superTypes" : ["uima.cas.FSArray" ] }, 
    "ShortArray" : {
      "@id" : "uima.cas.ShortArray", 
      "@superTypes" : ["uima.cas.FSArray" ] }, 
    "Sofa" : {
      "@id" : "uima.cas.Sofa", 
      "@featureRefs" : ["sofaArray" ], 
      "@superTypes" : ["uima.cas.TOP" ] }, 
    "StringArray" : {
      "@id" : "uima.cas.StringArray", 
      "@superTypes" : ["uima.cas.FSArray" ] } }, 
  "@cas_feature_structures" : {
    "28" : {"@type" : "Sofa",  "sofaNum" : 1,  "sofaID" : "_InitialView" }, 
    "1" : {"@type" : "AllTypes",  "sofa" : 28,  "aBoolean" : true,  "aByte" : -117,  "aShort" : -112,  "aLong" : 1234,  "aFloat" : 1.3,  "aDouble" : 2.6,  "aString" : "some \"String\"",  "aFS" : 35,  "aArrayBoolean" : [false ],  "aArrayMrBoolean" : 74,  "aArrayMrByte" : 77,  "aArrayByte" : [0,  0 ],  "aArrayShort" : [ ],  "aArrayMrShort" : 80,  "aArrayString" : [null ],  "aArrayMrString" : 83,  "aListInteger" : [ ],  "aListString" : ["testStr" ],  "aListFs" : [0 ] }, 
    "83" : {"@type" : "StringArray",  "@collection" : [null ] }, 
    "80" : {"@type" : "ShortArray",  "@collection" : [ ] }, 
    "77" : {"@type" : "ByteArray",  "@collection" : [0,  0 ] }, 
    "74" : {"@type" : "BooleanArray",  "@collection" : [false ] }, 
    "35" : {"@type" : "AllTypes",  "sofa" : 28,  "aBoolean" : false } }, 
  "@cas_views" : {
    "28" : [1 ] } }