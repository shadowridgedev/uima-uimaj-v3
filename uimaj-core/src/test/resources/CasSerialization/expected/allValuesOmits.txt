{"@context" : {
    "AllTypes" : {
      "@id" : "org.apache.uima.test.AllTypes", 
      "@featureRefs" : ["sofa",  "aFS",  "aArrayMrBoolean",  "aArrayMrByte",  "aArrayMrShort",  "aArrayMrString",  "aListMrInteger",  "aListMrString",  "aListFs",  "aListMrFs" ], 
      "@superTypes" : ["uima.tcas.Annotation",  "uima.cas.AnnotationBase",  "uima.cas.TOP" ] }, 
    "BooleanArray" : {
      "@id" : "uima.cas.BooleanArray", 
      "@superTypes" : ["uima.cas.FSArray",  "uima.cas.ArrayBase",  "uima.cas.TOP" ] }, 
    "ByteArray" : {
      "@id" : "uima.cas.ByteArray", 
      "@superTypes" : ["uima.cas.FSArray" ] }, 
    "ShortArray" : {
      "@id" : "uima.cas.ShortArray", 
      "@superTypes" : ["uima.cas.FSArray" ] }, 
    "Sofa" : {
      "@id" : "uima.cas.Sofa", 
      "@featureRefs" : ["sofaArray" ], 
      "@superTypes" : ["uima.cas.TOP" ] }, 
    "StringArray" : {
      "@id" : "uima.cas.StringArray", 
      "@superTypes" : ["uima.cas.FSArray" ] } }, 
  "@cas_feature_structures" : [
    {"@id" : 28,  "@type" : "Sofa",  "sofaNum" : 1,  "sofaID" : "_InitialView" }, 
    {"@id" : 1,  "@type" : "AllTypes",  "sofa" : 28,  "aBoolean" : true,  "aByte" : -117,  "aShort" : -112,  "aLong" : 1234,  "aFloat" : 1.3,  "aDouble" : 2.6,  "aString" : "some \"String\"",  "aFS" : 35,  "aArrayBoolean" : [false ],  "aArrayMrBoolean" : 74,  "aArrayMrByte" : 77,  "aArrayByte" : ["AAA=" ],  "aArrayShort" : [ ],  "aArrayMrShort" : 80,  "aArrayString" : [null ],  "aArrayMrString" : 83,  "aListInteger" : [ ],  "aListString" : ["testStr" ],  "aListFs" : [0 ] }, 
    {"@id" : 35,  "@type" : "AllTypes",  "sofa" : 28,  "aBoolean" : false }, 
    {"@id" : 74,  "@type" : "BooleanArray",  "@collection" : [false ] }, 
    {"@id" : 77,  "@type" : "ByteArray",  "@collection" : ["AAA=" ] }, 
    {"@id" : 80,  "@type" : "ShortArray",  "@collection" : [ ] }, 
    {"@id" : 83,  "@type" : "StringArray",  "@collection" : [null ] } ], 
  "@cas_views" : {
    "28" : [1 ] } }